### ISSUE TITLE
It should be possible to interact/comment on each job

### ISSUE THREAD

**Issue by jancborchardt** [opensourcedesign/opensourcedesign.github.io#100]
Some big problems with the job board at the moment are:

- it’s very static, discussion is difficult to start
- you don’t know the state of it, if it’s taken already
- we don’t know if it’s done

As far as I know it’s possible to embed Discourse thread replies in a website. We could have a category in the Discourse called »Jobs« (you can’t create posts but will be led to the [job submission form](http://opensourcedesign.net/jobs/job-form/)), and one thread per job. Then on every job there’s the discussion thread below.
@erlend-sh could you point us how to do that maybe? :)

We do have to require that the people have to register at Discourse though and be subscribed to the issue. But maybe that can be automated and will lead to growth.

cc @jdittrich @simonv3 @belenbarrospena @janushead @guiguru @HeikoTietze @bnvk @evalica

**Comment by simonv3** [comment]
This looks like it's what we want: 

https://meta.discourse.org/t/embedding-discourse-comments-via-javascript/31963

**Comment by simonv3** [comment]
Okay! This has been implemented but it can use a _lot_ of improvement. 

I think a lot of that improvement will come from figuring out the formatting and the markdown, eg.:

https://discourse.opensourcedesign.net/t/jobs-libreoffice-mascot/328

http://opensourcedesign.net/jobs/jobs/2017-06-28-libreoffice-mascot

Yay \o/

cc @bnvk @jancborchardt @evalica

**Comment by bnvk** [comment]
w00t w00t, great @simonv3 great step and progress 👍 

For some reason I had to update the ID in your link to Discourse from 328 to 353, not sure why.

How does this currently get created? Is it 100% automated? Is that perhaps why the ID issue happened?

**Comment by simonv3** [comment]
yeah, the ID issue probably happened because I have since deleted and created that thing a couple of times for testing purposes. 

This is actually pretty incredible functionality from Discourse. We can set it so that stuff gets automatically gets created when Discourse polls the feed.xml for the jobs (in which case we can only do it for jobs I think, not for, say, articles), or that it gets created when someone views the page (as is the case now). 

As some of the old projects gets viewed this might take some curation on the forum, hiding posts etc.

**Comment by bnvk** [comment]
> we can only do it for jobs I think, not for, say, articles

Why is this? We can (and should / am working towards) have multiple RSS feeds.

**Comment by simonv3** [comment]
Looks like a limitation on the Discourse side. There's only one input field for where to fetch RSS feed data from and it seems to apply to all embedded Discourse threads.

If we have separate repos we can't do this I think, but isn't it also fairly standard to provide an RSS feed of all updates from a site?

**Comment by jancborchardt** [comment]
So we could have a unified feed and then a "category" attribute which would tell Discourse where to add the topic?

**Comment by simonv3** [comment]
I think so? I mean, by no means am I an expert on this.

**Comment by marielle-G15** [comment]
We could have a category in the Discourse called »Jobs« (you can’t create posts but will be led to the job submission form), and one thread per job. Then on every job there’s the discussion thread below.

**Comment by jancborchardt** [comment]
@marielle-G15 sorry about this outdated issue – this is actually something that is working already exactly as you describe :)
- Category in the Discourse: https://discourse.opensourcedesign.net/c/job-postings
- Corresponds with the list at https://opensourcedesign.net/jobs/
- Discussion from the forum shows in a job posting: https://opensourcedesign.net/jobs/jobs/2019-03-19-design-of-badges-for-different-xmpp-compliance-levels

One thing we could additionally do is show the amount of comments in the list on the job board directly, and highlight ones without comments? Is that something you would be interested in looking into @marielle-G15?

**Comment by Erioldoesdesign** [comment]
> One thing we could additionally do is show the amount of comments in the list on the job board directly, and highlight ones without comments?

I can look at this new aspect of this issue when look at the job page design 🙌
